lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -42,10 +42,14 @@ def drop
 
       # Set `site.url` and `site.baseurl` if unset.
       def add_url_and_baseurl_fallbacks!
-        site.config["url"] ||= Value.new("url", proc { |_c, r| r.url_without_path })
+        Jekyll::GitHubMetadata.log :debug, "Adding `url` and `baseurl` fallbacks..."
```

Comment:
These would be better in the `Value` `proc { }` block, otherwise the benefits of using a Value (the fact that computation is deferred) is lost.